topology/topology.c

```diff
@@ -417,6 +420,13 @@ int main(int argc, char *argv[])
 		case 'p':
 			pre_process_config = true;
 			break;
+		case 'l':
```

Comment:
right this is the option that should come from the conf file itself

Reply:
now in manifest...